packages/web-app-user-management/src/views/Groups.vue

```diff
@@ -0,0 +1,244 @@
+      @confirm="deleteGroups"
+    />
+    <main class="oc-flex oc-flex-column oc-height-1-1 oc-p-m">
+      <app-loading-spinner v-if="loadResourcesTask.isRunning" />
+      <template v-else>
+        <div class="oc-app-bar">
+          <div class="oc-flex oc-flex-between">
+            <oc-breadcrumb class="oc-flex oc-flex-middle" :items="breadcrumbs" />
+          </div>
+          <div class="oc-flex-1 oc-flex oc-flex-start oc-mt-m">
+            <div v-if="selectedGroups.length" class="oc-flex oc-flex-middle">
+              <span v-text="selectedGroupsText" />
+              <oc-button
+                id="files-clear-selection"
+                v-oc-tooltip="$gettext('Clear selection')"
+                :aria-label="$gettext('Clear selection')"
+                class="oc-ml-m"
+                appearance="outline"
+                @click="unselectAllGroups"
+              >
+                <oc-icon name="close" />
+              </oc-button>
+              <oc-button appearance="outline" class="oc-ml-m" @click="toggleDeleteGroupModal">
+                <oc-icon name="delete-bin" />
+                <translate>Delete</translate>
+              </oc-button>
+            </div>
+            <div v-else>
+              <oc-button variation="primary" appearance="filled" @click="toggleCreateGroupModal">
+                <oc-icon name="add" />
+                <translate>New group</translate>
+              </oc-button>
+            </div>
+          </div>
+        </div>
+        <no-content-message
+          v-if="!groups.length"
+          id="user-management-groups-empty"
+          class="files-empty"
+          icon="group-2"
+        >
+          <template #message>
+            <span v-translate>No groups in here</span>
+          </template>
+        </no-content-message>
+        <div v-else>
+          <GroupsList
+            :groups="groups"
+            :selected-groups="selectedGroups"
+            class="oc-mt-m"
+            @toggleSelectGroup="toggleSelectGroup"
+            @toggleSelectAllGroups="toggleSelectAllGroups"
+          />
+        </div>
+      </template>
+    </main>
+  </div>
+</template>
+
+<script>
+import GroupsList from '../components/Groups/GroupsList.vue'
+import CreateGroupModal from '../components/Groups/CreateGroupModal.vue'
+import DeleteGroupModal from '../components/Groups/DeleteGroupModal.vue'
+import AppLoadingSpinner from 'web-pkg/src/components/AppLoadingSpinner.vue'
+import NoContentMessage from 'web-pkg/src/components/NoContentMessage.vue'
+import { useStore } from 'web-pkg/src/composables'
+import { ref } from '@vue/composition-api'
+import { clientService } from 'web-pkg/src/services'
+import { useTask } from 'vue-concurrency'
+import { bus } from 'web-pkg/src/instance'
+import { mapActions } from 'vuex'
+
+export default {
+  components: {
+    GroupsList,
+    AppLoadingSpinner,
+    NoContentMessage,
+    CreateGroupModal,
+    DeleteGroupModal
+  },
+  setup() {
```

Comment:
why do we mix vues options api and composition api for a newly created component, i would love to have it composition api style only :)